geohash.py

```diff
@@ -75,12 +82,12 @@ def _encode_i2c(lat,lon,lat_length,lon_length):
 	return ret[::-1]
 
 def encode(latitude, longitude, precision=12):
-	if latitude >= 90.0 or latitude < -90.0:
+	# Check Latitude
+	if latitude > 90.0 or latitude < -90.0:
 		raise Exception("invalid latitude.")
-	while longitude < -180.0:
-		longitude += 360.0
-	while longitude >= 180.0:
-		longitude -= 360.0
+
+	# Wrap Longitude
+	longitude = (longitude + 180.0) % 360.0 - 180.0

```

Comment:
The loss of precision isn't related to this change. This change was just to make the computation a 1-liner for cleaner code.